docs/OncoTree-Mapping-Tool.md

```diff
@@ -184,3 +188,26 @@ After filling in the `ONCOTREE_CODE` column with an OncoTree code for each sampl
 |P16|S16|65|MNM|
 |P17|S17|66|MYCF|
 |P18|S18|66|RBL|
+
+# Ontology Mapping Tool
+
+The Ontology Mapping tool was developed to facilitate the mapping between different cancer classification systems. We currently allow the mappings between OncoTree, ICD-O, NCIt, UMLS and HemeOnc systems. The tool and the mapping file can be found [here](https://github.com/cBioPortal/oncotree/tree/master/scripts)
```

Comment:
This sounds good .. but this PR should also contain the script as well (ontology_mapping_tool.py) (or .. ontology_to_ontology_mapping_tool.py) before it gets merged.

Reply:
I will ask @rmadupuri to add it

Reply:
@sheridancbio @ritikakundra added the script and renamed it to `ontology_to_ontology_mapping_tool.py`